app/scripts/store/dimStoreBucket.directive.js

```diff
@@ -130,6 +131,15 @@
     vm.moveDroppedItem = dimActionQueue.wrap(function(item, equip, $event, hovering) {
       var target = vm.store;
 
+      if (target.id === _.sortBy(dimStoreService.getStores(), 'lastPlayed').reverse()[0].id && equip) {
+        dimInfoService.show('doubleclick', {
```

Comment:
As a note for the future, we may want to extend infoservice to automatically suppress repeated "tip" dialogs even if folks don't click "don't show this again".